components/ActionBar/Article.js

```diff
@@ -1,46 +1,50 @@
 import React, { Component, Fragment, useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
 import ActionBar from './'
 import DiscussionIconLink from '../Discussion/IconLink'
 import { getDiscussionIconLinkProps } from './utils'
 
-import { fontStyles } from '@project-r/styleguide'
+import { fontStyles, button, plainButtonRule } from '@project-r/styleguide'
```

Comment:
```suggestion
import { fontStyles, plainButtonRule } from '@project-r/styleguide'
```
The lower case button is just the primitiv web button that does not need to be imported.